crates/uv-cli/src/lib.rs

```diff
@@ -2174,6 +2174,14 @@ pub struct BuildArgs {
     #[arg(long, overrides_with("build_logs"), hide = true)]
     pub no_build_logs: bool,
 
+    /// Always build through PEP 517, don't use the fast path for the uv build backend.
+    ///
+    /// By default, uv won't create a PEP 517 build environment for packages using the uv build
+    /// backend, but use a fast path that calls into the build backend directly. This option forces
+    /// always using PEP 517.
+    #[arg(long)]
+    pub no_fast_path: bool,
```

Comment:
I think we should give this a more descriptive name, like `--use-external-uv` or something. Since we might add other "fast paths" in the future. What do you think?

Reply:
`--force-pep517` seems reasonable...

Reply:
Ref #9600